packages/3d-web-client-core/src/rendering/composer.ts

```diff
@@ -91,11 +91,17 @@ export class Composer {
   private readonly gaussGrainPass: ShaderPass;
 
   private ambientLight: AmbientLight | null = null;
+  private environmentConfiguration: EnvironmentConfiguration;
 
   public sun: Sun | null = null;
   public spawnSun: boolean;
 
-  constructor(scene: Scene, camera: PerspectiveCamera, spawnSun: boolean = false) {
+  constructor(
+    scene: Scene,
+    camera: PerspectiveCamera,
+    spawnSun: boolean = false,
+    environmentConfiguration: EnvironmentConfiguration,
+  ) {
```

Comment:
If the 2 comments above make sense, then we can alter the constructor signature to:
```
constructor(
    scene: Scene,
    camera: PerspectiveCamera,
    spawnSun: boolean,
    private environmentConfiguration?: EnvironmentConfiguration | undefined,
  )
  ```